css/style.css

```diff
@@ -116,10 +145,246 @@ body {
 }
 
 .my-recent-works {
-  padding-top: 5px;
+  padding-top: 10px;
   text-align: center;
   padding-left: 2%;
   padding-right: 2%;
-  padding-bottom: 30px;
-  line-height: 2rem;
 }
+
+.my-recent-works h3 {
+  line-height: 3rem;
+  font-weight: 700;
+  font-size: 40px;
+}
+
+.my-recent-works img {
+  margin-top: 20px;
+}
+
+/* ---------------------mobile version------------ */
+
+.my-works h1 {
+  margin-bottom: 20px;
+  margin-top: 50px;
+  text-align: center;
+  font-size: 30px;
+  font-weight: bold;
+  line-height: 1.2em;
+}
+
+.my-works {
+  max-width: 375px;
+  text-align: center;
+}
+
+.my-works .indicator {
+  margin-bottom: 10px;
+  text-align: center;
+}
+
+.works {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 20px;
+  padding: 10px;
+}
+
+.work-1,
+.work-2,
+.work-3,
+.work-4,
+.work-5,
+.work-6 {
+  border-radius: 8px;
+  border: 1px solid #d0d9d4;
+  display: grid;
+  grid-template-columns: 1fr;
+  align-items: center;
+  grid-template-rows: 1fr;
+  justify-content: center;
+  grid-auto-rows: 50%;
+  padding-bottom: 15px;
+}
+
+.works .img-holder {
+  width: 100%;
+  height: 220px;
+  border-bottom: none;
+  padding: 0;
+  background-color: #ebf0ee;
+}
+
+.posts .works-button {
+  padding: 10px;
+  color: #fff;
+  margin: 30px 0;
+  outline: none;
+  border: none;
+  background: #36b37f;
+  border-radius: 4px;
+  font-size: 20px;
+}
+
+.posts h2 {
+  margin: 15px;
+  margin-top: 10px;
+  font-size: 25px;
+  font-weight: 500;
+  font-weight: bold;
+}
+
+.about-me {
+  max-width: 375px;
+  text-align: center;
+  padding: 10px;
+}
+
+.about-me-text {
+  width: 327px;
+  padding: 10px;
+}
+
+.about-me-text h2 {
+  padding-top: 30px;
+  font-size: 40px;
+  font-weight: 700;
+}
+
+.about-me .about-me-text p {
+  text-align: center;
+  line-height: 1.5em;
+  font-weight: 400;
+  font-style: normal;
+}
+
+.frameworks h2,
+.skills h2,
+.languages h2 {
+  margin-top: 30px;
+  margin-bottom: 25px;
+  text-align: center;
+}
+
+.about-me .about-me-text h2 {
+  margin: 20px auto;
+}
+
+.about-me .get-resume {
+  padding: 15px;
+  color: #fff;
+  margin: 20px 0;
+  outline: none;
+  border: none;
+  background: #36b37f;
+  border-radius: 4px;
+  font-weight: 500;
+  font-size: 18px;
+}
+
+.my-knowledge {
+  display: grid;
+  grid-template-columns: 1fr;
+  gap: 10px;
+  width: 375px;
+}
+
+.my-knowledge .flex-languages ul {
+  display: flex;
```

Comment:
- [ ] You did an amazing job styling the cards for knowledge but kindly move them more to the center because in the right it has more space than on the left. Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/159127033-acbfe612-d40e-4333-8a3b-c2ee34c09105.png)

Reply:
Hi, Here I have added all the constructive comments from you guys.